tests/integration/targets/vmware_ops_deploy_ovf_test/vars/main.yml

```diff
@@ -5,9 +5,9 @@ deploy_ovf_hostname: "{{ vcenter_hostname }}"
 deploy_ovf_username: "{{ vcenter_username }}"
 deploy_ovf_password: "{{ vcenter_password }}"
 deploy_ovf_validate_certs: false
-deploy_ovf_datacenter_name: Eco-Datacenter
-deploy_ovf_cluster_name: Eco-Cluster
-deploy_ovf_folder: /Eco-Datacenter/vm/e2e-qe
+deploy_ovf_datacenter_name: "{{ vcenter_datacenter }}"
+deploy_ovf_cluster_name: "{{ vcenter_cluster_name }}"
+deploy_ovf_folder: "/{{ vcenter_datacenter }}/vm/e2e-qe"
```

Comment:
I think the job does not recognize the var here due to the '/' or something else
```
TASK [vmware_ops_deploy_ovf_test : Create Test VM] *****************************
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: /{{ vcenter_datacenter }}/vm/e2e-qe: 'vcenter_datacenter' is undefined\n\nThe error appears to be in '/home/kni/actions-runner-vmware-ops/eco-vcenter/cloud.vmware_ops/cloud.vmware_ops/ansible_collections/cloud/vmware_ops/tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/main.yml': line 6, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  block:\n    - name: Create Test VM\n      ^ here\n"}
TASK [vmware_ops_deploy_ovf_test : Destroy Test VM] ****************************
ok: [localhost]
PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
NOTICE: To resume at this test target, use the option: --start-at vmware_ops_deploy_ovf_test
FATAL: Command "./runme.sh" returned exit status 2.
Test: vmware_ops_deploy_ovf_test Failed  
```

Reply:
It's failing because you import this variable after in the playbook. The failing step is the setup step [here](https://github.com/redhat-cop/cloud.vmware_ops/pull/72/files#diff-7e0af2df845248be0892611082f7bbb70a9bfa5be25ffefe3585d7e7e1294b22L6-L21) and the variables are included [here](https://github.com/redhat-cop/cloud.vmware_ops/pull/72/files#diff-7e0af2df845248be0892611082f7bbb70a9bfa5be25ffefe3585d7e7e1294b22R49)